init_and_run_app.sh

```diff
@@ -0,0 +1,40 @@
+#! /usr/bin/env bash
+
+# NAME: init_and_run_app.sh
+#
+# BRIEF: This script installs packages needed in the environment to run the
+# See Stuff application and then runs it locally.
+
+check_current_directory () {
+  if [[ ! -d '../rails_see_stuff' ]]; then
+    echo 'You must be in the directory, rails_see_stuff, after cloning the code'
+    echo 'from https://github.com/kimlew/rails_see_stuff'
+    exit 1
+  fi
+}
+
+check_if_rbenv_installed () {
+  if ! command -v rbenv > /dev/null; then
+    echo 'You must install the package manager, rbenv. For example, install with:'
+    echo 'sudo apt install rbenv OR'
+    echo 'brew install rbenv ruby-build'
+    exit 1
+  fi
+}
+
+check_current_directory
+check_if_rbenv_installed
+
+# Use specific version of Ruby for project that works with Rails 7.
```

Comment:
Suggestion: at this point it might be a good idea to put in the command

```
set -e
```

so that for the rest of your script if any of the commands fail (have a non-zero exit code) your script will exit with the command's exit code.

Reply:
I forgot that that is always a good idea. Added.